README.md

```diff
@@ -59,34 +96,85 @@ Throughput numbers from these scripts with various different configuration setti
 | | | 4096 | BF16/FP8 | 12,500 TPS | `Llama3-8B.py` | `--train_module.float8_config.enabled=true` |
 | **13B** | OLMo-1124 | 4096 | BF16 | 4,600 TPS | `OLMo2-13B.py` | |
 | | | 4096 | BF16/FP8 | 5,500 TPS | `OLMo2-13B.py` | `--train_module.float8_config.enabled=true` |
+| **32B** | OLMo-0325 | 4096 | BF16 | 1,500 TPS | `OLMo2-32B.py` | |
+| | | 4096 | BF16/FP8 | 1,800 TPS | `OLMo2-32B.py` | `--train_module.float8_config.enabled=true` |
```

Comment:
This is not accurate. I think the 1,800 number was for BF16, and we never officially ran the 32B with FP8.